### PR TITLE
Fixed AppVeyor failing tests

### PR DESCRIPTION
**Pull request by alpaix**
`Test.Utility` consumes `Microsoft.VisualStudio.ProjectSystem.Interop`
that propogates its interop dll to test projects and overrides the correct
vs14/vs15 signed interop dll. This caused strong-name verification failure
when loading test assemblies.

To fix that I changed the dependency type to `build` (non-transitive).

Also excluded `NuGet.CommandLine.Tests` from running on AppVeyor CI due to
their flakiness. The key issue was inability of AppVeyor to capture
console output.

According to Feodor:

> AppVeyor "console" is not the same as running an app in your local
> console. In some cases AppVeyor can't catch the output, especially if the
> app uses direct console API.

The workaround would be redirecting the output to file(s) and read it
later.

### CODE REVIEW

**Comment by alpaix** [comment]
//cc @joelverhagen @rohit21agrawal @emgarten @drewgil 

This change would make build status _happy_ green again:
[Build status](https://ci.appveyor.com/project/NuGetTeam/nuget-client)

File: test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetInitCommandTests.cs [hunk 1]
```diff
@@ -547,18 +547,11 @@ public void InitCommand_Fail_SourceIsInvalid()
             var invalidPath = "foo|<>|bar";
             using (var testInfo = new TestInfo(invalidPath, TestFileSystemUtility.CreateRandomTestFolder()))
             {
-                var args = new string[]
```
Comment (joelverhagen) [inline test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetInitCommandTests.cs:550]: Why the changes here?

**Comment by joelverhagen** [comment]
:shipit:

File: test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetInitCommandTests.cs (hunk 1 above)
Comment (alpaix) [inline test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetInitCommandTests.cs:550]: needed to quote arguments
